operations/bills.md

```diff
@@ -132,7 +132,12 @@ Extent of data to be returned. E.g. it is possible to specify that together with
           "BillingCode": "Billing code value",
           "Name": "Acme, Inc.",
           "FiscalIdentifier": "Fiscal identifier",
-          "AdditionalTaxIdentifier": "Additional tax identifier"
+          "AdditionalTaxIdentifier": "Additional tax identifier",
```

Comment:
Would be better for the user if possible to fill these with real life examples rather than "null".

Reply:
Agreed, but let's do it as a follow up.

Reply:
https://github.com/MewsSystems/gitbook-connector-api/pull/764